ui/src/pageLayout/containers/Nav.tsx

```diff
@@ -76,7 +76,7 @@ class SideNav extends PureComponent<Props, State> {
     const tokensLink = `${orgPrefix}/load-data/tokens`
     const clientLibrariesLink = `${orgPrefix}/load-data/client-libraries`
     // Settings
-    const settingsLink = `${orgPrefix}/settings/members`
+    const settingsLink = `${orgPrefix}/settings/members` // Might be different due to React Router. See src/index.tsx
```

Comment:
```suggestion
    const settingsLink = `${orgPrefix}/settings` // Might be different due to React Router. See src/index.tsx
```
I haven't tested this.

Reply:
ohhhh I get what you were saying now! I'll check it out and try to break it.

Reply:
Couldn't get this to work with Cloud off. It goes makes clicking the wrench go to `settings/variables/` when it should go to `settings/members`

Reply:
I think what you had was best? Does this work the other way around?